website/cue/reference/components/sources/base/internal_logs.cue

```diff
@@ -6,19 +6,21 @@ base: components: sources: internal_logs: configuration: {
 			Overrides the name of the log field used to add the current hostname to each event.
 
 			By default, the [global `log_schema.host_key` option][global_host_key] is used.
+			Set to `""` to suppress this key.
```

Comment:
If this was intended to be on a separate line, you'll need an additional linebreak in the rust comment

Reply:
ahh good. catch. I did intend that to be on a separate line. That is good to know.